build/dockerfiles/rhel.install.sh

```diff
@@ -7,8 +7,20 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
+       microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all; \
+elif [ $architecture == "s390x" ]; then \
+       microdnf install -y git findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
+       for i in g/glibc-common-2.29-9 g/glibc-all-langpacks-2.29-9 g/glibc-2.29-9 o/oniguruma-6.9.1-2 j/jq-1.6-2 s/skopeo-0.1.36-6.dev.gitd93a581 c/containers-common-0.1.36-6.dev.gitd93a581; do wget https://rpmfind.net/linux/fedora-secondary/releases/30/Everything/s390x/os/Packages/$i.fc30.s390x.rpm; done; \
```

Comment:
See comments on https://github.com/eclipse/che-plugin-registry/pull/428/files#r410271403 TL;DR why hardcode versions? is there a newer skopeo >=0.1.40?

Reply:
@nickboldt Please check updated comment on https://github.com/eclipse/che-plugin-registry/pull/428#discussion_r412089434

Reply:
commented over there. can you rebase this PR so it is mergeable?

Reply:
@nickboldt @benoitf I've updated PR, please review it.